x-pack/platform/plugins/private/data_visualizer/public/application/data_drift/charts/overlap_distribution_chart.tsx

```diff
@@ -61,8 +62,10 @@ export const OverlapDistributionComparison = ({
       />
 
       <Settings
-        // TODO connect to charts.theme service see src/plugins/charts/public/services/theme/README.md
-        baseTheme={LEGACY_LIGHT_THEME}
+        baseTheme={chartBaseTheme}
+        theme={{
+          axes: { gridLine: { horizontal: { visible: false }, vertical: { visible: false } } }, // Hide grid lines
```

Comment:
Nit: With `LEGACY_LIGHT_THEME` it seems the default line width for the area chart is thinner than with the updated theme (can be seen in the "Comparison" columns charts of the screenshots I shared in the other comment about grid lines.